doc/manuals/jx/jx.md

```diff
@@ -568,7 +568,27 @@ If multiple clauses are specified, they are evaluated from left to right.
 [[i, j] for i in range(5) for j in range(4) if (i + j)%2 == 0]
 = [[0, 0], [0, 2], [1, 1], [2, 0], [2, 2], [3, 1]]
 ```
-    
+
+### Anaphoric Expressions
```

Comment:
Anaphoric is a fifty-cent word that most people won't recognize.  Maybe explain this as "using functions like methods" or something along those lines?